spec/features/users/user_edit_spec.rb

```diff
@@ -9,11 +9,11 @@
       visit manage_email_path
     end
 
-    scenario 'user sees error message if form is submitted without email', :js, idv_job: true do
+    scenario 'user is not able to submit form without entering an email' do
       fill_in 'Email', with: ''
       click_button 'Update'
 
-      expect(page).to have_content t('valid_email.validations.email.invalid')
+      expect(page).to have_current_path manage_email_path
```

Comment:
Modern browsers have built-in validation for email fields and won't allow you to submit the form unless the field contains a string that looks like an email address.